src/NServiceBus.Core.Tests/Msmq/MsmqMessageSenderTests.cs

```diff
@@ -30,19 +33,16 @@ public void Should_set_label_when_convention_configured()
             {
                 DeleteQueue(path);
                 CreateQueue(path);
-                var messageSender = new MsmqMessageSender(new FakeContext())
-                {
-                    Settings = new MsmqSettings(),
-                    MessageLabelConvention = _ => "mylabel",
-                };
+                var messageSender = new MsmqMessageSender(new MsmqSettings(), _ => "mylabel");
+
                 var bytes = new byte[]
                 {
                     1
                 };
                 var headers = new Dictionary<string, string>();
                 var outgoingMessage = new OutgoingMessage("1", headers, bytes);
-                var sendOptions = new TransportSendOptions(queueName);
-                messageSender.Send(outgoingMessage, sendOptions);
+                var dispatchOptions = new DispatchOptions(new DirectToTargetDestination(queueName),new AtomicWithReceiveOperation(), new List<DeliveryConstraint>() );
```

Comment:
See comment above, not sure what the gain is for just the tests. The transport default is now a explicit concept on the transport definition